examples/protocol_specification_ex/sample.yaml

```diff
@@ -5,10 +5,10 @@ license: Apache 2.0
 description: 'A protocol for negotiation over a fixed set of resources involving two parties.'
 speech_acts:
   cfp:
-    query: DataModel
+    query: "DataModel"
   propose:
-    query: DataModel
-    price: float
+    query: "DataModel"
+    price: "pt:optional[pt:dict[DataModel, pt:bool] or pt:str or pt:dict[pt:int, Unit] or pt:set[pt:int] or pt:dict[_IOTApp, pt:bytes] or pt:list[Ali] or pt:bool]"
```

Comment:
I don't think we should permit class names starting with an underscore. See here: https://visualgit.readthedocs.io/en/latest/pages/naming_convention.html, the convention is UpperCaseCamelCase (see regex here: https://regexr.com/2rilj)

Reply:
See my first reply.

Reply:
Done